src/resolvers.ts

```diff
@@ -54,12 +56,15 @@ export const resolvers = {
       if (!passwordMatch) {
         throw new UnauthorizedError('Credenciais inválidas');
       }
-
-      const token = Jwt.sign({ userId: user.id }, process.env.JWT_TOKEN as string, { expiresIn: '1h' });
+      const rememberMe = args.input.rememberMe;
+      //ternário = if else
+      const expiresIn = rememberMe ? '7d' : '1d';
+      const token = Jwt.sign({ userId: user.id }, process.env.JWT_TOKEN as string, { expiresIn: expiresIn });
+      const refreshToken = Jwt.sign({ userId: user.id }, process.env.JWT_TOKEN as string, { expiresIn: '7d' });
 
       //faz com que retorne no playground
       //retorna o usuário por completo pq está pegando direto do banco
-      return { user: user, token: token };
+      return { user, token, rememberMe: refreshToken };
```

Comment:
não precisa retornar `rememberMe`
```suggestion
      return { user, token };
```